### ISSUE TITLE
Metacarpels Incorrectly rig as Proximal joints on hands

### ISSUE THREAD

**Issue by InsaneGrox** [Neos-Metaverse/NeosPublic#1782]
## Describe the bug
Currently, even if named "Meta" or "Metacarpel", Metacarpels on hands currently do not set up correctly, and instead rig as Proximal joints, causing weird as hell Hand movement & bends that should not be possible.

## Relevant issues
I believe in the process of fixing #1691 a side effect was introduced where Metacarpels get detected as proximal bones even if specifically named "meta" or "metacarpel"

## To Reproduce
[Steps to reproduce the behavior:
1. Import Nearly any avatar with rigged metacarpels
2. Notice how the hands are all screwed up and mangled and notice how they are assigned as proximal bones.

## Expected behavior
Metacarpels to be recognized correctly and function as such.

## Screenshots / Video
Will post later when I get around to it.

## Bug information (please complete the following information):
 - How often does it happen: Always
 - Does the bug persist after restarting Neos? Yes
 - Neos Version: 2021.2.22.855
 - Neos Platform: Windows
 - Link to reproduction World/Item: N/A, caused by import.
 - Regression: Yes, Not sure when it happened, but I believe it worked just fine in 2021.1.29.612 and before.

## Additional context
Uh, try and make it so that it can tell the difference between bad rigging and actual metacarpels? idk, Maybe require "Meta" to be in the name, or detect how close the root of the bone is to the wrist root.

## Reporters:
N/A

**Comment by Frooxius** [comment]
Do you have any sample model that we could use for testing? Or at least visual of what the sample hierarchy looks like?

Fixing problems like this is difficult in vacuum, as it can easily break other models that worked before, as they come in all kinds of rigging conventions (or lack of).

**Comment by InsaneGrox** [comment]
![image](https://user-images.githubusercontent.com/54233743/109021513-f308c480-7688-11eb-89be-7918415c3f63.png)
First bones are all named as such, Finger_Meta_Side, rest of the bones are named Finger_Number_Side, example chain would be 
Index_Meta_Left, Index_1_Left, Index_2_Left, Index_3_Left, I'm not sure how this could be fixed without undoing the fix from before, though if a model has 4 joints for each fingers and the first is not a metacarpel, or the last is not an _end bone added by FBX, that seems like bad rigging on the modeler's part, this particular model also has an issue where the middle finger isn't automatically detected, but that's a seperate issue entirely, and the metacarpal issue occurs even with that finger removed, also occurs on several other models I have tried importing, so it's not this model specifically.

**Comment by InsaneGrox** [comment]
![2021-02-28 17 40 13](https://user-images.githubusercontent.com/54233743/109436149-2b433680-79ec-11eb-891d-cedce7a40275.jpg)

**Comment by chemicalcrux** [comment]
I am observing this with my avatar. It was previously detected correctly; now, the bones are off by one, using the metacarpal as the proximal, and so forth.

**Comment by Frooxius** [comment]
Can you show me the hierarchy structure of these along with their full names please?

**Comment by chemicalcrux** [comment]
Here's the hierarchy in Blender. Every bone is weighted.

![image](https://user-images.githubusercontent.com/8015368/111057852-75023700-8458-11eb-9858-758a33d26b28.png)

I can share my avatar file privately if you're unable to replicate it with something you have access to.

**Comment by AegisTheWolf** [comment]
Here is another example of a rig incorrectly assigning the metacarpal bones as proximal and ignoring the distal bones
![image](https://user-images.githubusercontent.com/65252793/112231281-6a883000-8c04-11eb-8ca2-6be61e760847.png)

**Comment by AegisTheWolf** [comment]
Upon further experimentation I did discover this could relate to another issue regarding tooltip sizes being inconsistent with the avatar creator, The tooltip size is effected by the middle finger size. After scaling the avatar creator hands, hand bones, and rig scale to no effect I then remembered about this issue of the avatar creation not recognizing the metacarpals on my rig, therefore Neos thinks the metacarpals is the Proximal bones thus Changing the size of the middle finger and the corresponding tooltip anchor. (I removed the metacarpals and reimported and setup the rig and it does indeed scale the tooltip correctly. I hope this info helps.
